lib/main.js

```diff
@@ -12,6 +12,10 @@ const DEFAULT_DYNAMIC_REPLACE_INTERVAL_MS = 2000;
 
 const DISABLED_MSG = "⛔️";
 
+// Override DISABLED_MSG badge text (uncomment to use)
+//const DISABLED_MSG = "↻";
+
```

Comment:
what is purpose of this part?

Reply:
its for dev testing of symbols, I was thinking of using different symbols for the extension.